core/provider/service.go

```diff
@@ -260,24 +260,31 @@ func (s *Service) FetchResources(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-
 	failedProviders := map[string]error{}
+	resourcesCount := 0
+	upsertedResourceCount := 0
 	for _, p := range providers {
 		startTime := time.Now()
 		s.logger.Info(ctx, "fetching resources", "provider_urn", p.URN)
-		resources, err := s.getResources(ctx, p)
+		newResourcesCount, resources, err := s.fetchNewResources(ctx, p)
 		if err != nil {
 			s.logger.Error(ctx, "failed to get resources", "error", err)
 			continue
 		}
+		resourcesCount += newResourcesCount
+		upsertedResourceCount += len(resources)
```

Comment:
```suggestion
		totalFetchedResourcesCount += newResourcesCount
		updatedResourcesCount += len(resources)
```
this would give more understanding in the underlying value